connectors/sources/zoom.py

```diff
@@ -0,0 +1,542 @@
+        return await self.api_client.fetch(url=url)
+
+    async def get_past_meeting_participants(self, meeting_id):
+        url = APIS["PAST_MEETING_PARTICIPANT"].format(
+            base_url=BASE_URL, meeting_id=meeting_id, page_size=MEETING_PAGE_SIZE
+        )
+        async for participants in self.api_client.scroll(url=url):
+            for participant in participants.get("participants", []) or []:
+                yield participant
+
+    async def get_recordings(self, user_id):
+        # Zoom recording does not retrieve data that is more than 4 months old.
+        end_date = datetime.utcnow()
+        for _ in range(4):
+            start_date = end_date + timedelta(days=-30)
+            url = APIS["RECORDING"].format(
+                base_url=BASE_URL,
+                user_id=user_id,
+                date_from=format_recording_date(date=start_date),
+                date_to=format_recording_date(date=end_date),
+                page_size=MEETING_PAGE_SIZE,
+            )
+            async for recordings in self.api_client.scroll(url=url):
+                for recording in recordings.get("meetings", []) or []:
+                    yield recording
+            end_date = start_date
+
+    async def get_channels(self, user_id):
+        url = APIS["CHANNEL"].format(
+            base_url=BASE_URL, user_id=user_id, page_size=CHAT_PAGE_SIZE
+        )
+        async for channels in self.api_client.scroll(url=url):
+            for channel in channels.get("channels", []) or []:
+                yield channel
+
+    async def get_chats(self, user_id, chat_type):
+        # Zoom chat does not retrieve data that is more than 6 months old.
+        end_date = datetime.utcnow()
+        start_date = end_date + timedelta(days=-180)
+        url = APIS["CHAT"].format(
+            base_url=BASE_URL,
+            user_id=user_id,
+            chat_type=chat_type,
+            date_from=format_chat_date(date=start_date),
+            date_to=format_chat_date(date=end_date),
+            page_size=CHAT_PAGE_SIZE,
+        )
+        async for chats in self.api_client.scroll(url=url):
+            for chat in chats.get("messages", []) or []:
+                yield chat
+
+    async def get_file_content(self, download_url):
+        return await self.api_client.content(url=download_url)
+
+
+class ZoomDataSource(BaseDataSource):
+    name = "Zoom"
+    service_type = "Zoom"
+
+    def __init__(self, configuration):
+        super().__init__(configuration=configuration)
+        self.configuration = configuration
+
+    def _set_internal_logger(self):
+        self.client.set_logger(self._logger)
+
+    @cached_property
+    def client(self):
+        return ZoomClient(configuration=self.configuration)
+
+    @classmethod
+    def get_default_configuration(cls):
+        return {
+            "account_id": {
+                "label": "Account ID",
+                "order": 1,
+                "type": "str",
+                "value": "",
```

Comment:
We can remove `value` fields entirely now.
```suggestion
```

Reply:
But leave the `fetch_past_meeting_details` value as it's a bool type, and bool types still need an initial value.